packages/adapters-library/src/adapters/prices-v2/products/usd/pricesV2UsdAdapter.ts

```diff
@@ -103,17 +105,19 @@ export class PricesV2UsdAdapter implements IPricesAdapter {
   }
 
   private async getSpotPriceByAddress({
-    tokenAddresses,
+    tokenAddress,
     vsCurrency = 'usd',
   }: GetSpotPriceByAddressInput): Promise<SpotPrice | null> {
-    const url = `${BASE_URL}/v1/chains/${this.chainId}/spot-prices/${tokenAddresses}`
+    const url = `${BASE_URL}/v1/chains/${this.chainId}/spot-prices/${
+      tokenAddress === E_ADDRESS ? ZERO_ADDRESS : tokenAddress
+    }`
```

Comment:
Just changing it at url level. The logs will display both the original token address and the url we are calling.